tools/webpack.assets.json

```diff
@@ -3,6 +3,14 @@
     "api": "./static/js/portico/api.js",
     "katex": ["./node_modules/katex/dist/katex.min.js"],
     "landing-page": "./static/js/portico/landing-page.js",
+    "signup": [
+               "./static/js/portico/signup.js",
+               "./node_modules/jquery-validation/dist/jquery.validate.js"
```

Comment:
@pweaver Since it was in `dist` I assumed that it was a minified version. Don't know how I ignored the fact that this file was mentioned in `source_filenames` which means that it was not minified. Replaced it with the minified version in dist folder.

>Did you test with production assets?

I have checked using the `--minify` option and it is working fine!